### ISSUE TITLE
Oxyplot won't install in Xamarin WP8 project

### ISSUE THREAD

**Issue by BelgianRockfan** [oxyplot/oxyplot#451]
I have a Xamarin project with a PCL part, an Android project and a Windows Phone 8.0 part.
I've got oxyplot working fine on Android, but when I try to deploy the project to a Windows Phone it gives me the following error: 
Cannot resolve reference assemblies. Please check the reference assemblies. Cannot resolve dependency to assembly 'Xamarin.Forms.Platform.WP8, Version=1.3.3.0, Culture=neutral, PublicKeyToken=null' because it has not been preloaded. When using the ReflectionOnly APIs, dependent assemblies must be pre-loaded or loaded on demand through the ReflectionOnlyAssemblyResolve event.

It throws this in MainPage.xaml and App.xaml. Does anyone know what would cause this, because I'm all out of ideas and I really don't want to use the XLabs charts.

Edit: If I use older versions of both Oxyplot and Xamarin.Forms, it gives the same error message but with a different version number for the missing assembly.

**Comment by objorke** [comment]
We updated the source code to Xamarin Forms 1.4.1 recently, could this be a version issue? I think the WP8 package (build 840) is up to date, but not the Xamarin.Forms package (build 804).
@mattleibow We need #274 :-)

**Comment by BelgianRockfan** [comment]
Im not sure it's a version issue, because I tried doing it with an older version of Xamarin.Forms, and I made sure everything from OxyPlot was on version 804.

**Comment by tibel** [comment]
The package _OxyPlot.Xamarin.Forms 2015.1.804-alpha_ has some issues.
This one is similar to #412 and #438.

**Comment by BelgianRockfan** [comment]
So any other version should work? I'll try it out and I'll let you know if that fixes it.

**Comment by tibel** [comment]
@BelgianRockfan I don't thinkt that there is ANY working version on nuget.

**Comment by BelgianRockfan** [comment]
Damn, is the issue being worked on? Can you give me a rough ETA? Like I said, I would really like to use this, because I haven't found any decent free alternative.

**Comment by tibel** [comment]
It should be fixed in the sources. So you can compile it by your own.
But no nuget packages so far :cry:

**Comment by objorke** [comment]
I just updated the iOS, Android, Mac, Mobile and XamarinForms packages to the latest source code.
The iOS, Android and Xamarin.Forms assemblies were built on a PC. The Mac package was built on a Mac, of course. I used the latest stable Xamarin release.
I got some error messages for Windows Universal when building the OxyPlot.Mobile package, need to investigate that closer...

**Comment by BelgianRockfan** [comment]
Well, it builds on both WP and Android, which is an improvement, but it doesn't render on Windows Phone. Is it possible me being on WP 8.1 might be an issue?

**Comment by tibel** [comment]
@BelgianRockfan No, WP81 (Silverlight) should work.

Have a look at the sample:
https://github.com/oxyplot/oxyplot/tree/develop/Source/Examples/Xamarin.Forms/SimpleDemo/SimpleDemo.WinPhone

This should work, but haven't tested right now.

**Comment by objorke** [comment]
The SimpleDemo is currently a Windows Phone 8.0 project. I have tested it and could not find any issues. I also changed target to Windows Phone Silverlight 8.1, and it still works.

I also tested the OxyPlot.Xamarin.Forms 2015.1.840-alpha package and found no issues.
I created both a portable and a shared Xamarin.Forms app. Referenced Xamarin.Forms 1.4.1.6349. Tested both Windows Phone 8.0 and Windows Phone Silverlight 8.1
